deepchem/utils/fake_data_generator.py

```diff
@@ -0,0 +1,157 @@
+"""
+A fake data generator
+"""
+import random
+import numpy as np
+from deepchem.data import NumpyDataset
+from deepchem.feat import GraphData
+
+
+class FakeGraphGenerator:
```

Comment:
@arunppsg and I discussed offline. We think it makes sense to keep public and add to docs since we are making some improvements beyond the torch geometric